responsive-web-design-piano/css/style17.css

```diff
@@ -0,0 +1,48 @@
+/* box model reset */
+html {
+    box-sizing: border-box;
+}
+
+
+*, *:before, *:after {
+    box-sizing: inherit;
+}
+
+
+/* piano styles */
+#piano {
+    width: 992px;
+    height: 290px;
+    margin: 80px auto;
+    padding: 90px 20px 0 20px;
+    background-color: #00471b;
+}
+
+
+.keys {
+    width: 949px;
+    height: 180px;
+    padding-left: 2px;
+    background-color: #040404;
+}
+
+
+.key {
+    position: relative;
+    width: 41px;
+    height: 175px;
+    margin: 2px;
+    float: left;
+    background-color: #ffffff;
+}
+
+
+/* pseudo elements */
+.key.black--key::after {
+    content: "";
+}
+
+/* 
+    Give `.key.black--key::after` a `position` set to `absolute`
```

Comment:
Give `.key.black--key::after` a `position` set to `absolute`
and a `left` property with a value `-18px`.